torch/jit/annotations.py

```diff
@@ -49,7 +44,8 @@ class EvalEnv(object):
 
     def __init__(self, rcb):
         self.rcb = rcb
-        if torch.distributed.rpc.is_available():
+        if torch.distributed.is_available():
```

Comment:
any reason for just checking distributed instead of RPC? They should be the same for now, but we might change build script to have dedicated flag for RPC later?